<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNTWindowClass</class>
 <widget class="QWidget" name="CNTWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CNT</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Information</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Max Depth:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nodes:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Inner Nodes:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="maxDepthLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="nodeCountLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="innerNodeCountLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>BoundingBox:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="gridStyle">
            <enum>Qt::DashLine</enum>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>57</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <row>
            <property name="text">
             <string>Low</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Up</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>X</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Y</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Z</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>252</red>
               <green>175</green>
               <blue>62</blue>
              </color>
             </brush>
            </property>
           </item>
           <item row="0" column="1">
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>114</red>
               <green>159</green>
               <blue>207</blue>
              </color>
             </brush>
            </property>
           </item>
           <item row="0" column="2">
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>173</red>
               <green>127</green>
               <blue>168</blue>
              </color>
             </brush>
            </property>
           </item>
           <item row="1" column="0">
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>252</red>
               <green>175</green>
               <blue>62</blue>
              </color>
             </brush>
            </property>
           </item>
           <item row="1" column="1">
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>114</red>
               <green>159</green>
               <blue>207</blue>
              </color>
             </brush>
            </property>
           </item>
           <item row="1" column="2">
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>173</red>
               <green>127</green>
               <blue>168</blue>
              </color>
             </brush>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Max Depth:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="depthLabel">
        <property name="text">
         <string>MaxDepth</string>
        </property>
        <property name="buddy">
         <cstring>maxDepthSlider</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="maxDepthSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="View3DWidget" name="openGLWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>View3DWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>widgets/View3DWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>maxDepthSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>depthLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
